src/atomate2/forcefields/schemas.py

```diff
@@ -0,0 +1,167 @@
+"""Job to prerelax a structure using an MD Potential."""
+
+import logging
+from typing import List
+
+from pydantic import BaseModel, Field
+
+# from jobflow import Maker, job
+from pymatgen.core.structure import Structure
+
+
+class InputDoc(BaseModel):
+    """The inputs used to run this job."""
+
+    structure: Structure = Field(None, description="The inputted structure.")
+    relax_cell: bool = Field(
+        None,
+        description="Whether cell lattice was allowed to change during relaxation.",
+    )
+    relax_kwargs: dict = Field(
+        None,
+        description="Keyword arguments that passed to :obj:`StructOptimizer.relax`.",
+    )
+    optimizer_kwargs: dict = Field(
+        None,
+        description="Keyword arguments that passed to :obj:`StructOptimizer`.",
+    )
+
+
+class OutputDoc(BaseModel):
+    """The outputs of this job."""
+
+    structure: Structure = Field(None, description="The final, relaxed structure.")
+
+    energy: float = Field(None, description="Total energy in units of eV.")
+
+    energy_per_atom: float = Field(
+        None,
+        description="Energy per atom of the final structure in units of eV/atom.",
+    )
+
+    forces: List[List[float]] = Field(
+        None,
+        description="The force on each atom in units of eV/A for the final structure.",
+    )
+
+    # NOTE: units for stresses were converted to kbar
+    # (* -10 from standard output) to comply with MP convention
+    stress: List[float] = Field(
+        None, description="The stress on the cell in units of kbar (in Voigt notation)."
+    )
+
+    trajectory: dict = Field(
+        None, description="Step-by-step trajectory of the structural relaxation."
+    )
+
+    steps: int = Field(
+        None, description="total number of steps needed to relax the structure."
+    )
+
+
+class FFStructureRelaxDocument(BaseModel):
+    """Document containing information on structure relaxation using a force field."""
+
+    structure: Structure = Field(
+        None, description="Final output structure from the task"
+    )
+
+    input: InputDoc = Field(
+        None, description="The inputted information used to run this job."
+    )
+
+    output: OutputDoc = Field(
+        None, description="The outputted information from this relaxation job."
+    )
+
+    MD_potential: str = Field(
+        None,
+        description="name of the interatomic potential used for relaxation.",
+    )
+
+    MD_potential_version: str = Field(
+        None,
+        description="version of the interatomic potential used for relaxation.",
+    )
+
+    @classmethod
+    def from_chgnet_result(
+        cls,
+        input_structure: Structure,
+        relax_cell: bool,
+        relax_kwargs: dict,
+        optimizer_kwargs: dict,
+        result: dict,
+        keep_info: list,
```

Comment:
Re-order and provide a default value. Also, do you actually need the input structure, won't this just be the first structure in the trajectory?

Lastly, based on my comment above about changing `trajectory` -> `ionic_steps`, the `keep_info` could be modified to remove the plurals in some cases (because we now have a list of dicts). Also `keep_info` could be renamed to `ionic_step_data`.

```suggestion
        result: dict,
        relax_kwargs: dict = None,
        optimizer_kwargs: dict = None,
        keep_info: tuple = ("energy", "forces", "stress", "magmoms", "structure"),
```

Reply:
RE: keeping the input structure--I was just including it to be in-line with the VASP TaskDoc (in case users wanted an easy-to-reach initial structure). It is indeed a duplicate, but at the same time so is having an `output.structure` and `output.output.structure` field in the VASP TaskDoc. Let me know your thoughts--I don't feel too strongly one way or the other.

Reply:
Sorry for the confusion, the structure should definitely be in the task document. But you don't need to pass it separately to this function. It can be found in the result already. So just extract it from there rather than requiring it to be passed manually.

Reply:
Ah, okay that makes sense